### PR TITLE
[DEV APPROVED] 10468 footer changes

### PR DESCRIPTION
**Pull request by andreferreiradlw**
**Target Process ticket**

[TP 10468](https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=userstory/10468/silent)

**Summary**

Changes made in the secondary footer and copyright message in order to match it with the footer in the MAS website. Some general styles shared between the two footers were moved to yeast in [PR 52](https://github.com/moneyadviceservice/yeast/pull/52).

### CODE REVIEW

File: app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss [hunk 1]
```diff
@@ -1,86 +1,26 @@
...
+  .icon--accessibility {
+    background-position: -20px 1px; 
   }
-}
-
-.footer__copyright{
-  margin: 0 30px;
-  padding-bottom: 10px;
-}
+}
```
Comment (margOnline) [inline app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss]: need a new line here

File: app/assets/stylesheets/layouts/_footer.scss [hunk 2]
```diff
@@ -190,3 +190,14 @@
...
+.footer__copyright {
+  line-height: 1.2rem;
+  margin-bottom: 16px;
+
+  @include column(12);
+
+  @include respond-to($mq-s) {
+    @include body(16, 22);
+  }
+}
```
Comment (margOnline) [inline app/assets/stylesheets/layouts/_footer.scss]: and here

File: app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss (hunk 1 above)
Comment (andreferreiradlw) [reply app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss]: done

File: app/assets/stylesheets/layouts/_footer.scss (hunk 2 above)
Comment (andreferreiradlw) [reply app/assets/stylesheets/layouts/_footer.scss]: and done

**Pull request merged**